packages/create-pantheon-decoupled-kit/src/index.ts

```diff
@@ -93,14 +107,20 @@ export const main = async (
 		const cmsType = isString(args.cmsType) ? args.cmsType.toLowerCase() : null;
 
 		if (cmsType) {
-			if (['wp', 'drupal', 'any'].indexOf(cmsType) == -1) {
+			if (
+				['wp', 'wordpress', 'drupal', 'd9', 'd10', 'any'].indexOf(cmsType) == -1
```

Comment:
```suggestion
				cmsTypeOptions.indexOf(cmsType) == -1
```
Now that this array is getting a bit long, maybe assign it to a variable right after the `if (cmsType) {` line?